api/sheets/places_in.md

```diff
@@ -1,58 +1,85 @@
 ---
 layout: default
-title: Places within a Place
+title: Retrieving Places Contained Within Another Place
 nav_order: 2
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Retrieve places contained within another place
+# Retrieving Places Contained Within Another Place
 
-Given a list of parent [`Place`](https://datacommons.org/browser/Place) [DCIDs](/glossary.html),
-(e.g. any [`State`](https://datacommons.org/browser/State), [`Country`](https://datacommons.org/browser/Country), etc.), return a list of child places
-contained within the specified DCIDs. Only returns children whose place type matches
-the request's `placeType` parameter.
+The `=DCPLACESIN(dcids, placeType)` formula returns lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html), such as [State](https://datacommons.org/browser/State), [Country](https://datacommons.org/browser/Country), and so on. It only returns children with a place type that matches the <code>placeType</code> parameter.
 
-## General information about this endpoint
+> **Note**
+> 
+> Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](/api/sheets/) before using this formula.
 
-**Formula**: `=DCPLACESIN(dcids, placeType)`
+## Formula {#formula}
 
-**Required arguments**:
+```
+=DCPLACESIN(dcids, placeType)
+```
 
-*   `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+## Required Arguments {#required-arguments}
 
-*   `placeType`: The type of the contained (child) `Place` nodes to filter by. For example, `City` and `County` are contained within `State`. For a
-    full list of available types, see [the Data Commons graph browser entry for `Place`](https://datacommons.org/browser/Place).
+* `dcids`: A list of (parent) `Place` nodes, identified by their DCIDs.
+* `placeType`: The type of the contained (child) `Place` nodes to filter by. For example,`City` and `County`are contained within `State`. For a full list of available types, see [the Data Commons graph browser entry for Place](https://datacommons.org/browser/Place).
 
+## Returns
 
-## Assembling the information you will need to use this formula
+Lists of child places from a list of parent [Place](https://datacommons.org/browser/Place) [DCIDs](https://docs.datacommons.org/glossary.html), such as [State](https://datacommons.org/browser/State), [Country](https://datacommons.org/browser/Country), and so on. It only returns children with a place type that matches the <code>placeType</code> parameter.
```

Comment:
Please rephrase this - State and Country are place types (not place dcid's). I think you want to say something like this: returns a list of child places of the specified place dcids, of the specified place type

Reply:
I changed it as you suggested.